tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -92,157 +257,91 @@ describe('amo/components/AddonReviewList', () => {
         { ...fakeReview, rating: 2 },
       ];
       const tree = render({ reviews });
-      const ratings = scryRenderedComponentsWithType(tree, Rating);
-      expect(ratings.length).toEqual(2);
+      const ratings = tree.find(Rating);
+      expect(ratings).toHaveLength(2);
 
-      expect(ratings[0].props.rating).toEqual(1);
-      expect(ratings[0].props.readOnly).toEqual(true);
-      expect(ratings[1].props.rating).toEqual(2);
-      expect(ratings[1].props.readOnly).toEqual(true);
+      expect(ratings.at(0).prop('rating')).toEqual(1);
```

Comment:
That's cool. I thought `toHaveProp()` was just a true/false thing to check if the prop exists.